.github/pull_request_template.md

```diff
@@ -7,3 +7,11 @@
 
 - [ ] Ensure to attach screenshots
 - [ ] Ensure to have UX team confirm screenshots
+
+*Only commit code on Fridays/before holidays after speaking with your team and pushing out a time sensitive fix*
+
+# After committing your change 
+
+- [ ] Validate that your changes have been pushed to stage and test
+- [ ] Manually deploy the prod-frontend-app-learner-portal-enterprise
+ pipeline in [GoCD](https://gocd.tools.edx.org/go/pipelines#!/) 
```

Comment:
[suggestion] Given GoCD is 2U-specific, I'm not sure we should be mentioned it directly in the `openedx` repo, given it's not applicable to any open source contributors/consumers of this repo. Might it make sense to say, "Ensure your changes are deployed to production" to keep it generic? Worth noting that not all contributors/consumers are guaranteed to have the same stage vs. prod type of environments as we do at 2U/edX. 

The "only commit code..." bit also seems 2U-specific (and even _team_ specific).